src/components/ImportExport.js

```diff
@@ -13,15 +13,16 @@ import List, { ListItem, ListItemText } from 'material-ui/List';
 import Radio from 'material-ui/Radio';
 import { FormGroup, FormControl, FormControlLabel, FormHelperText } from 'material-ui/Form';
 import downloadjs from 'downloadjs';
-import ImportExportIcon from '@material-ui/icons/ImportExport';
 import Tooltip from 'material-ui/Tooltip';
 import { getMapStyle, convertTrailFeaturesToDonuts } from '../utils/mapUtils';
 import { hexToRgb } from '../utils/convertToRGB'
 import { Trail, Hydrant } from '../utils/records';
 import Select from 'material-ui/Select';
 import { MenuItem } from 'material-ui/Menu';
 import Typography from 'material-ui/Typography';
-import {IconButton, InputLabel} from 'material-ui';
+import {IconButton, InputLabel, Input} from 'material-ui';
+import FileUpload from '@material-ui/icons/FileUpload';
+import OperationMessage from './OperationMessage';
```

Comment:
Let's rename this file to `ImportExportDialog` since it doesn't have the button or other aspects anymore.